docs/docs/dev-guide/service-preview.md

```diff
@@ -166,4 +167,4 @@ In this quick start, we're going to preview a change we make to the backend serv
 
 Service Preview will match HTTP headers based on the headers that are seen by the *sidecar* and not the edge gateway. Matches are made on the whole header, e.g., a match rule of `dev` will not match in the example above, while `/backend/dev` will match.
 
-While any HTTP header will match, in practice, using host-based routing (i.e., the `:authority` header), a custom HTTP header (e.g., the `x-service-preview` header used above), or an authentication header is recommended.
\ No newline at end of file
+While any HTTP header will match, in practice, using host-based routing (i.e., the `:authority` header), a custom HTTP header (e.g., the `x-service-preview` header used above), or an authentication header is recommended.
```

Comment:
The file was missing a trailing newline, and so Emacs (like any decent text editor) appended one when I saved.  But of course GitHub can't display what that difference is in a reasonable way, can it?